updater

```diff
@@ -433,49 +444,45 @@ Refreshes the Pi-Apps details for apps that do not need to be reinstalled for th
   
 }
 
-update_app() { #first arg is app name
+update_app() { #given app name, uninstall it, update app folder, install it again
```

Comment:
Its worth clearly documenting that "update_app" is only used for "real" updates, not refreshes anymore.

Reply:
Done in https://github.com/Botspot/pi-apps/pull/2708/commits/e7ae18a5cb75c9fc522ed832d00dfe03ae11a49c